proarc-webapp/src/main/resources/cz/cas/lib/proarc/webapp/client/ClientMessages.properties

```diff
@@ -226,6 +226,13 @@ DigitalObjectEditor_OptionalEditorAction_Title=Preview
 DigitalObjectEditor_OptionalEditorAction_Hint=Switch On/Off an alternative preview.
 DigitalObjectEditor_UnsupportedEditor_Msg=Choose another editor in the toolbar!
 
+DigitalObjectEditor_ExpandTree_Title=Expand...
```

Comment:
`...` sem nepatří. Viz. `Actions.asMenuItem` a parametr `willAsk`. Navíc v české lokalizaci je to naopak.